aws/terminator/storage_services.py

```diff
@@ -219,3 +219,129 @@ def created_time(self):
 
     def terminate(self):
         self.client.delete_backup_selection(BackupPlanId=self.plan_id, SelectionId=self.id)
+
+
+class MemoryDBClusters(Terminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, MemoryDBClusters, 'memorydb', lambda client: client.describe_clusters()['Clusters'])
+
+    @property
+    def id(self):
+        return self.instance["ARN"]
+
+    @property
+    def name(self):
+        return self.instance["Name"]
+
+    @property
+    def created_time(self):
+        return self.instance["CreationTime"]
+
+    def terminate(self):
+        self.client.delete_cluster(ClusterName=self.name)
+
+
+class MemoryDBACLs(DbTerminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, MemoryDBACLs, 'memorydb', lambda client: client.describe_acls()['ACLs'])
+
+    @property
+    def id(self):
+        return self.instance["ARN"]
+
+    @property
+    def name(self):
+        return self.instance["Name"]
+
+    @property
+    def ignore(self):
+        return self.name.startswith('default')
+
+    def terminate(self):
+        self.client.delete_acl(ACLName=self.name)
```

Comment:
```suggestion
    def terminate(self):
        try:
            self.client.delete_acl(ACLName=self.name)
        except botocore.exceptions.ClientError as ex:
            if not ex.response['Error']['Code'] == 'InvalidACLStateFault':
                raise
```

Reply:
Hi, based on the documentation, the delete_acl function does not throw the 'InvalidACLStateFault' exception.

Reply:
Using age_limit property instead.